cmd/kubeless/function/function.go

```diff
@@ -219,15 +223,22 @@ func getFunctionDescription(cli kubernetes.Interface, funcName, ns, handler, fil
 	}
 
 	resources := v1.ResourceRequirements{}
-	if mem != "" {
-		funcMem, err := parseMemory(mem)
+	if mem != "" || cpu != "" {
+		funcMem, err := parseResource(mem)
```

Comment:
I am not sure that I follow, are both `men` and `cpu` required? With this code if one of the two values are not set the function will fail (trying to parse an empty string). Is that expected?

Reply:
`parseResource` was changed to return an empty resources object if an empty string is passed as input - happy to change if you want but was the easiest way to get stuff working

Reply:
ah, true, I missed that. It's okay then.